### PR TITLE
Angular-Modal update activate method

### PR DESCRIPTION
**Pull request by btesser**
Support using locals within the activate method.
This is documented here https://github.com/btford/angular-modal\#modalserviceactivate

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:
If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/btford/angular-modal\#modalserviceactivate
- [x] Increase the version number in the header if appropriate. (Not appropriate)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }` (Not appropriate)

### CODE REVIEW

**Review by paullessing — APPROVED** [review]
Looks good to me

**Comment by btesser** [comment]
Fix faling tests

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR ready to merge pending express review by a maintainer.

**Pull request merged**
